functions/lib/functions/deployFunctionsToGCloud.sh

```
@@ -1,83 +1,87 @@
-#!/bin/bash
+#!/bin/
```

Shellcheck (0.11.0)

[error] This shebang specifies a directory. Ensure the interpreter is a file.

(SC2246)